app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -27,6 +30,10 @@
  */
 public class FirebaseTestsHelper {
 
+    // Same as MainScreenActivity for now since RecyclerView tests depend on it
+    // Will be refined as soon as task lists are linked to households
+    public static final String TEST_TASK_LIST_DOCUMENT_NAME = "85IW3cYzxOo1YTWnNOQl";
```

Comment:
Why so complicated tho 😆

Reply:
I kept the original name from MainScreenActivity - technically, this *could* be simplified & involves renaming the task list location on firestore

Reply:
As long as it is not generated automatically by firebase it's fine!

Reply:
[origin story]: One day, a HouseOrganizer dev was testing how to store TaskLists on firebase. At last, the method was successful and "85IW3cYzxOo1YTWnNOQl" was born. Legend has it that it's still being used by its creator as a hardcoded location

Reply:
Yup basically what I was thinking of when I saw this String 😆